_posts/2013-04-09-asset-compilation.md

```diff
@@ -29,15 +29,30 @@ production-env by default. You can pass custom options to the minifier via the
 you may supply a boolean value for `minifyJS:enabled`.
 
 For example, to disable minifying of CSS and JS, add in `ember-cli-build.js`:
-{% highlight bash %}
-minifyCSS: {
+{% highlight javascript %}
+app.options.minifyCSS.options = {
   enabled: false
-},
-minifyJS: {
+};
+
+app.options.minifyJS.options = {
```

Comment:
@rwjblue sure I can. But have a look at the issue I linked. You said this is currently broken? Is it fixed/ready to be in the docs?